site/frontend/src/pages/compare/compile/table/utils.ts

```diff
@@ -0,0 +1,14 @@
+import {ArtifactDescription} from "../../types";
+
+/**
+ * Returns a date in the past for which we want to display a historical chart.
+ */
+export function getDateInPast(artifact: ArtifactDescription): string {
```

Comment:
why is this taking an `ArtifactDescription`, instead of being called with the `artifact.date`?

Reply:
Well semantically it says "give me some date that is in the past of this artifact". But sure, it can take a date :)